packages/core/src/App/Containers/WarningCloseCreateRealAccountModal/warning-close-create-real-account-modal.jsx

```diff
@@ -1,39 +1,15 @@
 import React from 'react';
 import { connect } from 'Stores/connect';
-import { DesktopWrapper, MobileWrapper } from '@deriv/components';
 import { withRouter } from 'react-router';
 import WarningMessageModal from './warning-close-create-real-account-content';
 import './warning-close-create-real-account-modal.scss';
 
-const WarningCloseCreateRealAccountModal = ({
-    is_closing_create_real_account_modal,
-    setIsClosingCreateRealAccountModal,
-    closeRealAccountSignup,
-    routing_history,
-}) => (
-    <React.Fragment>
-        <DesktopWrapper>
-            <WarningMessageModal
-                is_closing_create_real_account_modal={is_closing_create_real_account_modal}
-                setIsClosingCreateRealAccountModal={setIsClosingCreateRealAccountModal}
-                closeRealAccountSignup={closeRealAccountSignup}
-                routing_history={routing_history}
-            />
-        </DesktopWrapper>
-        <MobileWrapper>
-            <WarningMessageModal
-                is_closing_create_real_account_modal={is_closing_create_real_account_modal}
-                setIsClosingCreateRealAccountModal={setIsClosingCreateRealAccountModal}
-                closeRealAccountSignup={closeRealAccountSignup}
-                routing_history={routing_history}
-            />
-        </MobileWrapper>
-    </React.Fragment>
-);
+const WarningCloseCreateRealAccountModal = props => <WarningMessageModal {...props} />;
```

Comment:
The `WarningMessageModal` implemented is identical for both `DesktopWrapper` and `MobileWrapper` so I removed as the wrappers & passed the props directly as they match the incoming props